### ISSUE TITLE
Clipping map plane

### ISSUE THREAD

**Issue by slukiceng** [tentone/geo-three#53]
Hi there I'm looking for a way to add the map to a plane and then clip the plane to a set of map bound coordinates (the idea being to be able to rotate and pan around an area and visualise some below ground data). Is this supported by the library, I've been trying out multiple approaches and getting nowhere?

**Comment by tentone** [comment]
This can be achieved using three.js clipping volumes. Define a box model to clip the map object.

https://threejs.org/examples/#webgl_clipping_advanced

**Comment by slukiceng** [comment]
Yeah I've been playing around using that example over the past fews days. I have the clipping volumes working. I know the issue is somewhere in the lines of code below, since when I uncomment the line of code at the bottom all the clipping is okay, just no map tiles are rendered on the plane. Any advice would be really appreciated.

      clipMaterial = new THREE.MeshPhongMaterial( {
        color: 0xee0a10,
        shininess: 100,
        side: THREE.DoubleSide,
        // Clipping setup:
        clippingPlanes: createPlanes( Planes.length ),
        clipShadows: true
      } );

      const provider = new Geo.OpenStreetMapsProvider();
      const map = new Geo.MapView(Geo.MapView.PLANAR, provider);
      map.updateMatrixWorld(true);
      map.material.clippingPlanes = createPlanes( Planes.length )
      map.material.clipShadows = true
      
      const geometry = new THREE.BoxGeometry( 0.7, 0.7, 0.7 );

      const mesh = map

     // This line of code below clips the plane exactly how I need, just not rendering the map tiles:
     // const mesh = new THREE.Mesh( map.geometry, clipMaterial );
